applications/FluidDynamicsApplication/custom_utilities/fluid_element_utilities.cpp

```diff
@@ -172,6 +172,55 @@ void FluidElementUtilities<TNumNodes>::SetTangentialProjectionMatrix(
     noalias(rTangProjMatrix) = id_matrix - outer_prod(rUnitNormal, rUnitNormal);
 }
 
+
+template < std::size_t TNumNodes>
+void FluidElementUtilities<TNumNodes>::DenseSystemSolve(
+    const boost::numeric::ublas::bounded_matrix<double,2,2> &rA,
+    const array_1d<double,2> &rB,
+    array_1d<double,2> &rX)
+{
+    boost::numeric::ublas::bounded_matrix<double,2,2> inverse;
```

Comment:
i would suggest to remove the boost::numeric::etc...

but it may well be a different PR

Reply:
why don't you use the one in math_utils btw?

Reply:
@RiccardoRossi I'm merging now to do this renaming all in one go ;)
Regarding the math utils thing, this is just adapted from my old element, which used its own function for some reason (I think back then there was no bounded matrix version of this function in math utils, but I might be wrong). I can refactor it as part of the porting to AMatrix.